aasemble/django/apps/buildsvc/tasks.py

```diff
@@ -9,6 +9,13 @@ def reprepro(repository_id, *args):
 
 
 @shared_task(ignore_result=True)
+def export(repository_id):
+    from .models import Repository
+    r = Repository.objects.get(id=repository_id)
```

Comment:
Don't think I understand this very well - I think you can only get a Repository _after_ it's saved. So this line will fail for new repositories.